source/_posts/2021-07-07-release-20217.markdown

```diff
@@ -455,11 +443,7 @@ If you are looking for a replacement, you can use the
   `websocket-client` library which is incompatible with the requirements of
   other integrations that are actively maintained.
 
-- It's not clear if this integration still works with the gpmdp app that now
-  only supports YouTube Music. If there's someone that uses the integration
-  successfully and wants to take on the maintenance task that is required to get
-  the integration in a compatible state, please create an issue to discus
-   the future of this integration.
+- It's not clear if this integration still works with the gpmdp app that now only supports YouTube Music. If there's someone that uses the integration successfully and wants to take on the maintenance task that is required to get the integration in a compatible state, please create an issue to discuss the future of this integration.
```

Comment:
Could you please keep the new lines in place, it is hard to find the change in this one now.